app/Services/AdventuresLeagueAdapter.php

```diff
@@ -83,33 +86,77 @@ public function getCharacter($filePath): ?Character
     private function getEntries($data, $characterId)
     {
         $entries = [];
+
+        // entries only begin on line 5 (index 4) of the exported csv
         for ($i = 4; $i < count($data); $i++) {
-            //Check if row is a magic item entry
             $isItemEntry = ($data[$i][0] == "MAGIC ITEM");
+            $isCharacterLogEntry = ($data[$i][0] == "CharacterLogEntry");
+            $isCampaignLogEntry = ($data[$i][0] == "CampaignLogEntry");
+            $isDmLogEntry = ($data[$i][0] == "DmLogEntry");
+            $isTradeLogEntry = ($data[$i][0] == "TradeLogEntry");
+            $isPurchaseLogEntry = ($data[$i][0] == "PurchaseLogEntry");
+
+            $isEntryLine = ($isCharacterLogEntry || $isDmLogEntry || $isCampaignLogEntry || $isTradeLogEntry || $isPurchaseLogEntry);
 
-            if (!$isItemEntry) {
+            if ($isEntryLine) {
+                $type = "";
+                if ($isDmLogEntry) {
```

Comment:
Oh i see, you need the `$isDmLogEntry` here, hmm ok, might be fine to ignore the above comment